pkg/clusterconfig/clusterconfig.go

```diff
@@ -108,7 +108,7 @@ func GetAWSConfig(listers *regopclient.Listers) (*Config, error) {
 	sec, err := listers.Secrets.Get(imageregistryv1.ImageRegistryPrivateConfigurationUser)
 	if err != nil && errors.IsNotFound(err) {
 		pollErr := wait.PollImmediate(1*time.Second, 5*time.Minute, func() (stop bool, err error) {
-			sec, err = client.Secrets(imageregistryv1.ImageRegistryOperatorNamespace).Get(cloudCredentialsName, metav1.GetOptions{})
+			sec, err = client.Secrets(installerConfigNamespace).Get(cloudCredentialsName, metav1.GetOptions{})
```

Comment:
@dmage it's actually the errors that are incorrect (https://github.com/openshift/cluster-image-registry-operator/blob/master/pkg/clusterconfig/clusterconfig.go#L122-L134), they should have the imageregistryv1.ImageRegistryOperatorNamespace instead of the installerConfigNamespace, as that was the old location for the kube-system/aws-creds secret that we no longer use.

Reply:
I misread the bug. Thanks, Corey. I've updated the PR.